docs/building-fcos.md

```diff
@@ -102,6 +102,7 @@ cosa() {
               --tmpfs /tmp -v /var/tmp:/var/tmp --name cosa                                         \
               ${COREOS_ASSEMBLER_CONFIG_GIT:+-v $COREOS_ASSEMBLER_CONFIG_GIT:/srv/src/config/:ro}   \
               ${COREOS_ASSEMBLER_GIT:+-v $COREOS_ASSEMBLER_GIT/src/:/usr/lib/coreos-assembler/:ro}  \
+              ${COREOS_ASSEMBLER_ADD_CERTS:+-v /etc/pki/ca-trust:/etc/pki/ca-trust:ro}              \
```

Comment:
```suggestion
              ${COREOS_ASSEMBLER_ADD_CERTS:+-v=/etc/pki/ca-trust:/etc/pki/ca-trust:ro}              \
```

Reply:
Without this, this fails for me in Zsh with:
```
COREOS_ASSEMBLER_ADD_CERTS=y
+cosa:16> podman run --rm -ti --security-opt 'label=disable' --privileged '--uidmap=1000:0:1' '--uidmap=0:1:1000' --uidmap 1001:1001:64536 -v /var/home/tim/projects/rhcos:/srv/ --device /dev/kvm --device /dev/fuse --tmpfs /tmp -v /var/tmp:/var/tmp '-v /etc/pki/ca-trust:/etc/pki/ca-trust:ro' quay.io/coreos-assembler/coreos-assembler:latest fetch
Error: creating named volume " /etc/pki/ca-trust": running volume create option: names must match [a-zA-Z0-9][a-zA-Z0-9_.-]*: invalid argument
+cosa:25> rc=125 
+cosa:25> set +x
```

As if it took that as a single string.

Reply:
Hmm, the other `-v` flags also don't use `=`. Do you know why this particular one fails?

Reply:
I'm OK doing this, but we should probably do it for all of them for consistency.

Reply:
I don't know and find it confusing as well. The only difference I can see is that in the other flags, we have shell variables expanded.

Reply:
OK, this looks ZSH specific :/

```
$ set -x
$ echo ${USER:+foo bar}
+/bin/zsh:10> echo 'foo bar'
foo bar
$ echo ${USER:+foo bar $USER}
+/bin/zsh:12> echo 'foo bar tim'
foo bar tim
$ bash
+/bin/zsh:13> bash
$ set -x
$ echo ${USER:+foo bar $USER}
+ echo foo bar tim
foo bar tim
$ echo ${USER:+foo bar}
+ echo foo bar
foo bar
```